app/locales/ro.json

```diff
@@ -0,0 +1,112 @@
+    "location_enabled_title": "Jurnalul Locațiilor Activat",
+    "maps_import_button_text": "Importa istoric locații",
+    "maps_import_disclaimer": "Safe Paths nu are nici o afiliere cu Google și nu dezvaluie NICIODATA datele dumneavoastra.",
+    "maps_import_text": "Pentru a verifica dacă v-ați intersectat cu cineva găsit pozitiv COVID-19 înainte de a fi descarcat aceasta aplicatie, puteți importa istoricul personal al locațiilor dumneavoastra.",
+    "maps_import_title": "Hărți Google",
+    "nCoV2019_url_info": "Pentru mai multe informații despre setul de date al acestei hărți",
+    "news_subtitle": "Cititi ultimele actualizări despre COVID-19 de la autoritatile de sănătate și alte surse.",
+    "news_title": "Ultimele știri",
+    "no_data": "Lipsa date",
+    "notification_2_weeks_ago": "Acum 2 săptămâni",
+    "notification_data_not_available": "Nu exista date expunere.",
+    "notification_select_authority": "Selectați Autoritate Sănătate",
+    "notification_title": "Profil Expunere 2 Săptămâni",
+    "notification_today": "astăzi",
+    "notification_warning_text": "Dacă exista o Autoritate de Sănătate în zona dumneavoastra, va puteți abona la aceasta pentru a obține actualizări regulate despre riscul de expunere.",
+    "notifications_exposure_format": "Acum {{daysAgo}} zile v-ați intersectat cu cineva infectat timp de {{exposureTime}} minute.",
+    "notifications_exposure_format_today": "Astăzi v-ați intersectat cu cineva infectat timp de {{exposureTime}} minute.",
+    "notifications_exposure_format_yesterday": "Ieri v-ați intersectat cu cineva infectat timp de {{exposureTime}} minute.",
+    "notifications_no_exposure": "Nici o expunere la COVID-19 în ultimele 2 săptămâni.",
+    "overlap_found_button_label": "Date Publice Incarcate",
+    "overlap_no_results_button_label": "Date Publice Incarcate",
+    "overlap_para_1": "Traseele verzi reprezinta istoricul dumneavoastra\n\nTraseele mov-deschis reprezinta istoricul obținut din datele publice.",
+    "overlap_title": "Verifica Intersectări",
+    "push_at_risk_message": "V-ați intersectat cu un pacient COVID-19",
+    "push_at_risk_title": "Factor de risc identificat",
+    "see_exposure_history": "Vezi istoricul expunerilor",
+    "settings_title": "Tablou de bord",
+    "share_location_data": "Trimite locațiile",
+    "show_overlap": "Accesează datele publice",
+    "team": "Echipa",
+    "team_para": "Echipa noastră este formata dintr-un consorțiu de epidemiologi, ingineri, oameni de stiință de date, promotori a sigurantei datelor personale, cat si profesori și cercetatori de la instituții cu reputație, ce includ: MIT, Harvard, Clinica Mayo, TripleBlind, EyeNetra, Ernst & Young și Link Ventures.",
+    "terms_of_use": "Termeni de folosire",
+    "terms_of_use_url": "https://docs.google.com/document/d/1mtdal_pywsKZVMXLHjjj5eKznipPLP8sM1HwFTIhjo0/edit#",
```

Comment:
remove all the 3 URL keys. they will fall back to the default.

Reply:
removed